Libraries/src/Amazon.Lambda.ApplicationLoadBalancerIdentity/Amazon.Lambda.ApplicationLoadBalancerIdentity.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Obviously, this will change if we put it in a separate repo and emphasize it is not Lambda specific.